aubase410/Package.md

```diff
@@ -53,37 +52,38 @@ This package includes, FHIR AUBase Resource types.
 | AUBaseProcedure | [[Definition]][s30] [[Ballerina Record]][m30] |
 | AUBaseEncounter | [[Definition]][s31] [[Ballerina Record]][m31] |
 
-[m1]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseComposition
-[m2]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationRequest
-[m3]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBasePathologyResult
-[m4]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBasePatient
-[m5]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticResult
-[m6]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBasePractitioner
-[m7]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseLocation
-[m8]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseHealthcareService
-[m9]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticImagingResult
-[m10]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedication
-[m11]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticReport
-[m12]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUMedicineList
-[m13]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticRequest
-[m14]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationAdministration
-[m15]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationStatement
-[m16]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseSpecimen
-[m17]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseBodyStructure
-[m18]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseMedicationDispense
-[m19]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseCondition
-[m20]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseDiagnosticImagingReport
-[m21]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUHealthProgramParticipationSummary
-[m22]: https://lib.ballerina.io/ballerinax/health.fhir.r4.aubase410/1.0.3/#AUBaseRelatedPerson
```

Comment:
Are these manually edited? The new links looks incorrect

Reply:
fixed